js/src/components/Footnotes.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Typography from '@material-ui/core/Typography';
+import Box from '@material-ui/core/Box';
+import { withStyles } from '@material-ui/core';
+import LessonGrid from '@anu/components/LessonGrid';
+import makeStyles from '@material-ui/core/styles/makeStyles';
+
+const StyledBox = withStyles((theme) => ({
+  root: {
+    marginBottom: theme.spacing(4),
+  },
+}))(Box);
+
+const useStyles = makeStyles((theme) => ({
+  divider: {
+    width: '100%',
+    height: '1px',
+    background: theme.palette.grey[300],
+    top: theme.spacing(3),
+  },
+}));
+
+const StyledTypography = withStyles((theme) => ({
+  root: {
+    '& a': {
```

Comment:
Why don't you use text formatting you have already defined in previous PRs?

Reply:
More things to override, like bullets styles, font size, etc. I think that for the particular type of paragraph easy to remove `...textStyle(theme),`.

Reply:
@adubovskoy if you wanted to override something for this paragraph, why didn't you have any styles from textStyle() defined here? Like spacing between paragraphs, etc?

Reply:
It really looks very different with inheriting styles, much more code to make it look as simple & laconic like the current implementation. I think we don't want to see the same big paragraph spacing here.